server.js

```diff
@@ -0,0 +1,14 @@
+var express = require("express");
+var app = express();
```

Comment:
use const instead for imported packages/objects if you're not intending to modify them